client/elements/sc-page-dictionary.html

```diff
@@ -268,42 +272,54 @@ <h3>Similar Spelling</h3>
 
           // Adding Adjacent Terms
           _computeAdjacentUrl() {
-              return `${this.$.meta.byKey('API_ROOT')}/adjacent`;
+              return `${this.$.meta.byKey('API_ROOT')}/dictionary_adjacent/${this.dictionaryWord}`;
           }
 
           _loadAdjacent() {
               this.$.adjacent_ajax.url = this._computeAdjacentUrl();
               return this.$.adjacent_ajax.generateRequest();
           }
 
-          _adjacentLoaded() {
-              console.log("loaded Adjacent");
-              console.log(this.AdjacentReturns);
-          }
-
-          _calculateClass(item) {
-              return (item === this.dictionaryWord) ? 'selected-terms-item' : '';
+          // Adding Similar Terms
+          _computeSimilarUrl() {
+              return `${this.$.meta.byKey('API_ROOT')}/dictionary_similar/${this.dictionaryWord}`;
           }
 
-          // Adding glossary results to related items drawer
-          _computeGlossaryUrl() {
-              return `${this.$.meta.byKey('API_ROOT')}/glossary`;
+          _loadSimilar() {
+              this.$.similar_ajax.url = this._computeSimilarUrl();
+              return this.$.similar_ajax.generateRequest();
           }
 
-          _loadGlossary() {
-              this.$.glossary_ajax.url = this._computeGlossaryUrl();
-              return this.$.glossary_ajax.generateRequest();
+          // Calculating if the current item is selected
+          _calculateClass(item) {
+              return (item === this.dictionaryWord) ? 'selected-terms-item' : '';
           }
 
+          // Converts glossary output to object
           _glossaryLoaded() {
-              console.log("loaded Glossary");
-              console.log(this._getGlossaryDescription("dhamma"));
-              console.log(this.glossaryReturns);
+              let result = {};
+              this.glossaryReturns[0].forEach((item) => result[item.word] = item.text);
+              this.glossaryList = result;
           }
 
-          _getGlossaryDescription(glossword) {
-              console.log(this.glossaryReturns(glossword));
-              return this.glossaryReturns(glossword);
+          _getGlossaryItems(glossaryList, inputArray, adjacent=false) {
+              if (inputArray) {
+                  let glossary = [];
+                  let glossaryObject = {};
+                  let glossText = '';
+                  if (glossaryList) {
+                      for (let glossWord in inputArray[0]) {
+                          let glossLookup = inputArray[0][glossWord];
+                          glossText = `<a href="/define/${glossLookup}">${glossLookup}</a>`;
+                          if (glossaryList[glossLookup]){
+                              glossText += `<em> (${glossaryList[glossLookup]})</em>`
+                          }
+                          glossaryObject = {"glossword": glossLookup, "glosstext": glossText};
+                          glossary.push(glossaryObject);
```

Comment:
1. Please use camelCase when naming object keys. The double quotes are not needed.
2. This can be inline:
```
glossary.push({ glossWord: glossLookup, glossText: glossText })